src/other/Table.js

```diff
@@ -138,8 +175,31 @@
     };
     
 
-    Table.prototype.click = function (d) {
-    };
+    Table.prototype.click = function (column) {
```

Comment:
This "click" method is already in use by the data row.  Can you rename it to "_headerClick" or such like (the "_" is to indicate its private).